CMakeLists.txt

```diff
@@ -42,7 +42,13 @@ endif()
 list(APPEND CMAKE_MODULE_PATH ${PROJECT_SOURCE_DIR}/cmake/modules)
 
 # Dependencies
-include(cmake/Dependencies.cmake)
+if(${ARCH} MATCHES "aarch64-linux")
+    include(cmake/Dependencies.aarch64-linux.cmake)
```

Comment:
inconsistent indentation.

Reply:
I ran "make lint" and did not see any errors being reported. I will fix this

Reply:
I will fix this

Reply:
fixed this